src/material/schematics/ng-update/upgrade-rules/package-imports-v8/secondary-entry-points-rule.ts

```diff
@@ -31,7 +37,7 @@ export class SecondaryEntryPointsRule extends MigrationRule<null> {
 
   // Only enable this rule if the migration targets version 8. The primary
   // entry-point of Material has been marked as deprecated in version 8.
-  ruleEnabled = this.targetVersion === TargetVersion.V8;
+  ruleEnabled = this.targetVersion === TargetVersion.V8 || this.targetVersion === TargetVersion.V9;
```

Comment:
I think it wouldn't be necessary to have `V8` here. This is because if someone updates to v8 using `ng update` it will actually install the v8 version of `@angular/material` and run those schematics.

Though, to keep our ng-update implementation work with `--from 7 --to 8`, I think we should keep it. I don't feel very strong about it though? Would it make things easier to just remove `v8` here?

Reply:
I don't feel strongly about it either, I just wasn't sure whether what I'm doing is correct.

Reply:
Yeah, both things would be totally correct. More ideal is keeping `v8` since it allows ´--from 7 --to 8`

Reply:
I think explicitly using v8 and and v9 is the right thing